examples/RAmPART_examples/selectRun.m

```diff
@@ -0,0 +1,1152 @@
+% This files purpose is to select a certain set of sampling options.MCMC from a
+% large pool of predefined examples. The input parameter b defines which of
+% the 2100 predefined option sets will be selected. A summary of the
+% selected sampling problems and algorithm:
+%
+%   1-100: Ring   + PT
+% 101-200: Ring   + RAMPART
+% 201-300: Gauss  + PT
+% 301-400: Gauss  + RAMPART
+% 401-500: Banana + PT
+% 501-600: Banana + RAMPART
+% 601-700: JakStat+ PT
+% 701-800: JakStat+ RAMPART
+% 801-900: mRNA   + PT
+% 901-1000: mRNA  + RAMPART
+% 1001-1100: RafMekErk + PT
+% 1101-1200: RafMekErk + RAMPART
+% 1201-1300: mRNA (exp) + PT
+% 1301-1400: mRNA (exp) + RAMPART
+% 1401-1500: Ring + RegionOnly
+% 1501-1600: Gauss + RegionOnly
+% 1601-1700: JakStat + RegionOnly
+% 1701-1800: mRNA (exp) + RegionOnly
+% 1801-1900: Bachmann + PT
+% 1901-2000: Bachmann + RAMPART
+% 2001-2100: Bachmann + RegionOnly
+% 
+% If two options.MCMC sets do possess the same sampling problem and algorithm,
+% they differ in their random seed. The results will be stored will running
+% in SAVE file. After the run terminated, a res_b.mat file will be created
+% and stores the final results.
+%
+% Written by benjamin ballnus (2017)
+
+function selectRun( b, targetDir )
```

Comment:
couldn't we make this a little bit more human readable? E.g. replace b by three algorithms: `example`, `method`, `index`.